### DISCUSSION TITLE
Azure4students : We could not confirm your status as a student

### DISCUSSION THREAD

**Discussion by Bunny7282927** [community/community#140678] [GitHub Education] [answered]
### Hi everyone,

Hi!

My GitHub student dev pack started 2 days ago, and I wanted to redeem some of the benefits including Azure.

In GitHub, I can access the student offers, and benefits, but when I try to activate them with the following link (https://aka.ms/azure4students) and select the GitHub method, I get this error:
![Screenshot_20241005-221523](https://github.com/user-attachments/assets/330a3346-e5ab-42ea-a5fe-313072aecee0)
We could not confirm your status as a student. Click here to begin the academic verification process. This process may take up to a few days. Once verification is complete, please come back and sign in with GitHub to redeem Azure for Students.

There are some screenshots:
![IMG_20241005_221811](https://github.com/user-attachments/assets/2449e29d-bd46-4b2e-bdc9-7bf9e3331ea8)
![IMG_20241005_221742](https://github.com/user-attachments/assets/0a71b391-b119-4fba-a7a2-1baae537ddd8)

And yes I'm using the email connected to my GitHub (it's the same) and I have tried to logout and relogin.

**Comment by davevad93** [comment, marked as answer]
Hi @Bunny7282927 ,

This is a known verification problem, #131336, #135739, #140395, to fix it you can do the following:

1. Contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your problem with Azure verification. 

Or

2.  Create a support ticket directly on [Azure](https://azure.microsoft.com/en-us/support/create-ticket).

I personally recommend the first option.
